<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>459</width>
     <height>147</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="ViewMemberInfoButton" >
        <property name="text" >
         <string>View/Edit member info</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="AddCommitteeButton" >
        <property name="text" >
         <string>Add new committee</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="AddMemberButton" >
        <property name="text" >
         <string>Add new member</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QPushButton" name="printPublicPhoneButton" >
        <property name="text" >
         <string>Print public phone list</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QPushButton" name="printPrivatePhoneButton" >
        <property name="text" >
         <string>Print private phone list</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="ViewCommitteesButton" >
        <property name="text" >
         <string>View/Delete committees</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ExitButton" >
      <property name="text" >
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
